cc3200-sys/src/simplelink.rs

```diff
@@ -217,15 +221,16 @@ c_like_enum! {
     }
 }
 
-// NetConfigGet only has an Id, and no opt, but we encode it the same way
-// as NetConfigSet for consistency.
+pub const SL_MAC_ADDR_LEN: usize = 6;
+
+// NetConfigGet only has an Id.
 c_like_enum! {
     NetConfigGet {
-        MacAddress = 0x0200,
-        Ipv4StaP2pClientGetInfo = 0x0301,
+        MacAddress = 0x02,
+        Ipv4StaP2pClientGetInfo = 0x03,
```

Comment:
It matches with this:
https://github.com/fabricedesre/cc3200-rs/blob/master/cc3200-sys/sdk/simplelink/include/netcfg.h#L78-L79
and the get-mac-address works.